<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCrashDISetting</class>
 <widget class="QWidget" name="WidgetCrashDISetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Item</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_Type">
        <property name="text">
         <string>crashDI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>id</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="customSpinBox" name="spinBox_Id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_Prompt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>func</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customComboBox" name="comboBox_func"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customComboBox" name="comboBox_type">
          <item>
           <property name="text">
            <string>碰撞开关(线段)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>inverse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="customCheckBox" name="checkBox_inverse">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>236</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
